src/renderer/getDiForUnitTesting.tsx

```diff
@@ -160,6 +161,7 @@ export const getDiForUnitTesting = (opts: { doGeneralOverrides?: boolean } = {})
       }));
     });
 
+    di.override(environmentVariablesInjectable, () => ({}));
```

Comment:
Instead of accumulating global overrides here, we can now use global override files.

Reply:
Resolved, this stuff will sort out eventually, bit by bit.